static/beta/stage/modules/fed-modules.json

```diff
@@ -985,6 +985,21 @@
             }
         ]
     },
+    "idmsvc": {
```

Comment:
Your appname is `hmsidm` in your source files here: https://gitlab.cee.redhat.com/identity-management/idmocp/idm-domains-frontend/-/blob/main/package.json#L75

The key has to match the `appname`. The same goes for the route to the manifest file.

Reply:
I see, then we will have to wait before we can merge.

Reply:
:+1: thanks mate for reviewing this and your time! if any further change could be necessary, more than happy to make it.